dashboard/test/models/school_test.rb

```diff
@@ -295,12 +283,8 @@ class SchoolTest < ActiveSupport::TestCase
   def clear_schools_and_dependent_models
     # Clear tables with hard dependencies (ie, MySQL foreign keys)
     # on the schools table.
-    Census::ApSchoolCode.delete_all
-    Census::IbSchoolCode.delete_all
     Census::CensusOverride.delete_all
```

Comment:
I'm seeing in the engineering plan that the `census_overrides` table will be removed, but I don't see mention of the model. Can this be deleted as well?

Reply:
great catch! removed

Reply:
Great! Changes looks good 👍